client/src/main/proto/spine/client/command_service.proto

```diff
@@ -30,18 +30,31 @@ package spine.client;
 import "spine/options.proto";
 
 option (type_url_prefix) = "type.spine.io";
-// We put gRPC-based classes into `grpc` sub-package, which is annotated as `@Internal`
-// to hide implementation details from the public API of the framework.
 option java_package = "io.spine.client.grpc";
 option java_multiple_files = true;
 option java_outer_classname = "CommandServiceProto";
 
 import "spine/core/command.proto";
 import "spine/core/ack.proto";
 
-// A service for sending commands from clients.
+// A service for posting commands to the application backend.
+//
 service CommandService {
 
-    // Request to handle a command.
+    // Posts the given command to the application backend.
+    //
+    // When the command is successfully received by the application, it responds with
+    // an acknowledgement. The received `Ack` may have one of the following statuses:
+    //
+    //    1. `Ok` meaning the command is accepted for further processing.
+    //    2. `Error` when a technical error occurs on the side of the application.
```

Comment:
Please be more specific. What is a technical error? What is the "side of the application"? Both client and server make the application as a whole.